halo/halo.py

```diff
@@ -166,6 +171,28 @@ def color(self, color):
         """
         self._color = color
 
+    @property
+    def placement(self):
+        """Getter for placement property.
+        Returns
+        -------
+        str
+            spinner placement
+        """
+        return self._placement
+
+    @placement.setter
+    def placement(self, placement):
+        """Setter for placement property.
+        Parameters
+        ----------
+        placement: str
+            Defines the placement of the spinner
+        """
+        if placement not in self.SPINNER_PLACEMENTS:
+            raise ValueError("unknown spinner placement '{0}', available are {1}".format(placement, self.SPINNER_PLACEMENTS))
```

Comment:
`unknown` -> `Unknown `

Reply:
Do you like the idea of raising ValueError on unknown placements?
The reason line 70 is `self.placement = placement` (not `self._placement = placement`) is because it calls this setter method which runs the snippet above.

I can simply remove the logic in the setter and say any placement that isn't `right` is assumed to be `left`.

Reply:
No, it makes sense to throw an error. This is correct.